test/e2e/run_generate_seccomp_test.go

```diff
@@ -0,0 +1,62 @@
+package integration
+
+import (
+	"io/ioutil"
+	"os"
+	"time"
+
+	. "github.com/containers/libpod/test/utils"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Podman generate profile", func() {
```

Comment:
"Podman run --generate-seccomp" to make it easier to understand what's being generated.

Reply:
I dislike having a dedicated flag for this. I'd prefer to trigger off a label that we can add when creating containers - makes this easier to use with other runtimes like CRI-O and Buildah.

Reply:
I thought we were going to do this via a label and not from podman at all.  That way anytool that creates a container with OCI spec containing the label would be able to generate the seccomp file.

Reply:
That's correct. We just use the flag to set the annotation whose value is a path to the seccomp profile. Other tools can do the same.

Reply:
If we use a label, we don't need a flag at all - just `podman run --label seccomp-generate:filename ...`

Reply:
We had to use `--annotation`. I still prefer having the --generate-seccomp one because it's easier to explore by users, supports bash completion etc.

Reply:
I'm a little reluctant because it couples us very closely to an OCI hook which we don't guarantee is installed

Reply:
That's a good point. I think we have a similar situation with `--init` but this doesn't mean we need to follow.

@weirdwiz @rhatdan @giuseppe, are you cool with following Matt's advice and use `--annotation` instead? I have no objection anymore and am cool with it.

Reply:
I am fine with this.